### ISSUE TITLE
Health check info in the Web UI

### ISSUE THREAD

**Issue by synotio** [containous/traefik#1783]
### Do you want to request a *feature* or report a *bug*?

Feature

### What did you do?

Configured healthchecks and Web UI

### What did you expect to see?

The current state of the backend health checks

### What did you see instead?

Only a list of the backends (the health section only shows status codes and response times).

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.3.1
Codename:     raclette
Go version:   go1.8.3
Built:        2017-06-16_11:21:48AM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
logLevel = "DEBUG"
debug = true

[docker]
domain = "traefik"
watch = true
swarmmode = true
filename = "/etc/traefik/docker.tmpl"

[web]
address = ":8080"
```

```
# /etc/traefik/docker.tmpl
# [...]
    [backends.backend-{{$backendName}}.healthcheck]
      path = "/"
      interval = "10s"
# [...]
```

CentOS 7, Docker 17.05.0-ce in swarm mode

### Details

Currently the health checks seem to operate mostly in silence. Some things that could be nice to have in the Web UI for each backend server:

* Current state (up/down)
* How many successful checks that have passed
* How many failures that have occurred
* Time of last successful check
* What polling interval and URL that's configured (this is actually present in /api/providers but not displayed)

**Comment by ldez** [comment]
Seems a duplicate of #180

**Comment by dfredell** [comment]
I agree it would be nice to see the backend status' in the UI. 
I did find that this data is available via the [metrics endpoint](https://docs.traefik.io/v1.7/configuration/metrics/). I'm using prometheus
```js
traefik_backend_server_up{backend="webserver",url="http://10.1.2.3:8080"} 1
traefik_backend_server_up{backend="webserver",url="http://10.1.2.4:8080"} 0
traefik_backend_server_up{backend="webserver",url="http://10.1.2.5:8080"} 1
```

**Comment by ldez** [comment]
Fixed in v2.0
